docs/cfn-template.yaml

```diff
@@ -0,0 +1,68 @@
+AWSTemplateFormatVersion: 2010-09-09
+Resources:
+  Queue:
+    Type: 'AWS::SQS::Queue'
+    Properties:
+      MessageRetentionPeriod: 300
+  QueuePolicy:
+    Type: 'AWS::SQS::QueuePolicy'
+    Properties:
+      Queues:
+        - !Ref Queue
+      PolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - events.amazonaws.com
+                - sqs.amazonaws.com
+            Action: 'sqs:SendMessage'
+            Resource: !GetAtt Queue.Arn
+  ASGTermRule:
+    Type: 'AWS::Events::Rule'
+    Properties:
+      EventPattern:
+        source:
+          - aws.autoscaling
+        detail-type:
+          - EC2 Instance-terminate Lifecycle Action
+      Targets:
+        - Id: 1
+          Arn: !GetAtt Queue.Arn
+  SpotTermRule:
+    Type: 'AWS::Events::Rule'
+    Properties:
+      EventPattern:
+        source:
+          - aws.ec2
+        detail-type:
+          - EC2 Spot Instance Interruption Warning
+      Targets:
+        - Id: 1
+          Arn: !GetAtt Queue.Arn
+  RebalanceRule:
+    Type: 'AWS::Events::Rule'
+    Properties:
+      EventPattern:
+        source:
+          - aws.ec2
+        detail-type:
+          - EC2 Instance Rebalance Recommendation
+      Targets:
+        - Id: 1
+          Arn: !GetAtt Queue.Arn
+  InstanceStateChangeRule:
+    Type: 'AWS::Events::Rule'
+    Properties:
+      EventPattern:
+        source:
+          - aws.ec2
+        detail-type:
+          - EC2 Instance State-change Notification
+      Targets:
+        - Id: 1
+          Arn: !GetAtt Queue.Arn
+Outputs:
```

Comment:
Can you please update the sections of the "[Infrastructure Setup](https://github.com/aws/aws-node-termination-handler#infrastructure-setup)" instructions that this template covers? It looks like you're capturing Step 1 and Step 3.

Reply:
@AustinSiu should remove cli commands for SQS and EventBridge Creation?

Reply:
We can leave the cli command references in the docs, for convenience. I was thinking more of adding a sentence pointing the reader to the template in the relevant. Something like the below to replace [this](https://github.com/aws/aws-node-termination-handler#1-create-an-sqs-queue):
```
Here is the AWS CLI command to create an SQS queue to hold termination events from ASG and EC2, although this should really be configured via your favorite infrastructure-as-code tool like CloudFormation or Terraform (template describing these resources can be found [here](url)):
```

Reply:
@AustinSiu changes are made as per request.

Anything else needs to change let me know.

Reply:
Thanks! Can you add it to "4. Create Amazon EventBridge Rules" as well?

Reply:
@AustinSiu done